src/test/java/org/grizz/factory/PlanetFactoryTest.java

```diff
@@ -0,0 +1,67 @@
+package org.grizz.factory;
+
+import lombok.extern.slf4j.Slf4j;
+import org.grizz.OgameCloneApplication;
+import org.grizz.config.security.SecurityConfig;
+import org.grizz.model.Building;
+import org.grizz.model.Planet;
+import org.grizz.model.User;
+import org.grizz.model.enummerations.BuildingType;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.*;
+
+@Slf4j
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {OgameCloneApplication.class, SecurityConfig.class})
+public class PlanetFactoryTest {
+
+    private static String NAME = "Planet Mother";
+
+    @Autowired
+    @InjectMocks
```

Comment:
You have no mocks and you actually not injecting mocks here (even if you had one). To make `InjectMocks` work you need to run test with `MockitoRunner` or use `initMocks` in `Before` method.